resources-compose/build.gradle.kts

```diff
@@ -61,3 +57,10 @@ kotlin {
         }
     }
 }
+
+afterEvaluate {
+    tasks.withType<JavaCompile>().configureEach {
```

Comment:
`withType` and `configureEach` is live collection processors, so `afterEvaluate` not required here